namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-incentives-prod/resources/variables.tf

```diff
@@ -49,6 +49,12 @@ variable "number_cache_clusters" {
   default = "2"
 }
 
-variable "rds_family" {
-  default = "postgres14"
```

Comment:
FYI: @ushkarev I inlined this variable as discussed as this value goes with the `db_engine_version` one.